test/js_interop_helpers_test/shared_tests.dart

```diff
@@ -4,6 +4,8 @@ library js_function_test;
 import 'dart:html';
 
 import 'package:js/js.dart';
+import 'package:js/js_util.dart' show newObject;
```

Comment:
so package:js/js_util.dart actually just exports `dart:js_util` so id say using it is better than depending on an external library. but you could also just keep using js_interop_helpers.dart until its fully removed too. your call.
```suggestion
import 'dart:js_util';
```